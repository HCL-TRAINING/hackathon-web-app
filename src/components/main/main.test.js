import React from "react";
import Main from "./main";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
// @ts-ignore
Enzyme.configure({ adapter: new Adapter() });
import renderer from "react-test-renderer";

describe("main component testing", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Main />);
    console.log("shallow debuger", wrapper.debug());
  });

  test("renders for snapshot", () => {
    const tree = renderer.create(<Main />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("renders main div", () => {
    expect(wrapper.find("#main").text()).toContain("main");
  });
});
