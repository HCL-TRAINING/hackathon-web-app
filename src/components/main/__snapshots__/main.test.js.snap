// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main component testing renders for snapshot 1`] = `
<div
  id="main"
>
  main
</div>
`;
